openapi.yaml

```diff
@@ -0,0 +1,675 @@
+        - name: instance_id
+          in: path
+          description: id of instance being deleted
+          required: true
+          schema:
+            type: string
+        - name: service_id
+          in: query
+          description: id of the service associated with the instance being deleted
+          required: true
+          schema:
+            type: string
+        - name: plan_id
+          in: query
+          description: id of the plan associated with the instance being deleted
+          required: true
+          schema:
+            type: string
+        - name: accepts_incomplete
+          in: query
+          description: asynchronous deprovision supported
+          schema:
+            type: boolean
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AsyncOperation'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '410':
+          description: Gone
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '422':
+          description: Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /v2/service_instances/{instance_id}/last_operation:
+    get:
+      summary: last requested operation state for service instance
+      tags:
+        - ServiceInstances
+      operationId: serviceInstance.lastOperation.get
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - name: instance_id
+          in: path
+          description: instance id of instance to find last operation applied to it
+          required: true
+          schema:
+            type: string
+        - name: service_id
+          in: query
+          description: id of the service being used
+          schema:
+            type: string
+        - name: plan_id
+          in: query
+          description: id of the plan being used
+          schema:
+            type: string
+        - name: operation
+          in: query
+          description: a provided identifier for the operation
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInstanceLastOperation'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
```

Comment:
I'm not 100% sure, but I think `error` and `description` field should be allowed to be returned with a 400

Reply:
I made an [issue](https://github.com/openservicebrokerapi/servicebroker/issues/403) to clear this up.

Reply:
👍